packages/pharos/src/components/combobox/pharos-combobox.ts

```diff
@@ -435,6 +448,13 @@ export class PharosCombobox extends ScopedRegistryMixin(FormMixin(FormElement))
     this.open = !this.open;
   }
 
+  private _normalizeString(textString: string) {
+    return textString
+      .normalize('NFKD')
+      .replace(/[\u0300-\u036f]/g, '')
```

Comment:
Might you leave a comment here explaining this? I don't know offhand what it does, myself! Is that a range of unprintable unicode characters?

Reply:
It's a range of all possible diacritics - after text is normalized, than we can remove the characters and compare the text.

Reply:
Please leave this as a code comment, if you don't mind!